tasks/managed/update-cr-status/update-cr-status.yaml

```diff
@@ -24,31 +24,106 @@ spec:
     - name: resource
       description: The namespaced name of the resource to be patched
       type: string
-    - name: resultsDirPath
+    - name: resultsDirName
       description: |
-        Path to the directory containing the result files in the data workspace which will be added to the
-        resource's status
+        Name of possible directories in subdirectory containing the result files in the data workspace 
+        which will be added to the resource's status
       type: string
+      default: "results"
+    - name: ociStorage
+      description: The OCI repository where the Trusted Artifacts are stored.
+      type: string
+      default: "empty"
+    - name: ociArtifactExpiresAfter
+      description: Expiration date for the trusted artifacts created in the
+        OCI repository. An empty string means the artifacts do not expire.
+      type: string
+      default: "1d"
+    - name: trustedArtifactsDebug
+      description: Flag to enable debug logging in trusted artifacts. Set to a non-empty string to enable.
+      type: string
+      default: ""
+    - name: orasOptions
+      description: oras options to pass to Trusted Artifacts calls
+      type: string
+      default: ""
+    - name: resultArtifacts
+      type: array
+      description: Array of artifacts to use to obtain results
+      default: []
+    - name: dataDir
+      description: The location where data will be stored
+      type: string
+      default: $(workspaces.data.path)
+    - name: taskGitUrl
+      type: string
+      description: The url to the git repo where the release-service-catalog tasks and stepactions to be used are stored
+    - name: taskGitRevision
+      type: string
+      description: The revision in the taskGitUrl repo to be used
   workspaces:
     - name: data
       description: Workspace where the results directory is stored
+  volumes:
+    - name: workdir
+      emptyDir: {}
+  stepTemplate:
+    volumeMounts:
+      - mountPath: /var/workdir
+        name: workdir
+    env:
+      - name: IMAGE_EXPIRES_AFTER
+        value: $(params.ociArtifactExpiresAfter)
+      - name: "ORAS_OPTIONS"
+        value: "$(params.orasOptions)"
+      - name: "DEBUG"
+        value: "$(params.trustedArtifactsDebug)"
   steps:
+    - name: skip-trusted-artifact-operations
+      ref:
+        resolver: "git"
+        params:
+          - name: url
+            value: $(params.taskGitUrl)
+          - name: revision
+            value: $(params.taskGitRevision)
+          - name: pathInRepo
+            value: stepactions/skip-trusted-artifact-operations/skip-trusted-artifact-operations.yaml
+      params:
+        - name: ociStorage
+          value: $(params.ociStorage)
+        - name: workDir
+          value: $(params.dataDir)
+    - name: use-trusted-artifact-array
```

Comment:
what is this doing? Also `params.resultArtifacts` isn't in the README

Reply:
This step tasks the `array-type` parameter and extract the artifacts.

it is the sister of the StepAction `use-trusted-artifact`

Reply:
fixed README too

Reply:
should it be in the stepactions dir and called via resolver then? I get it is the only task using this, but maybe better for consistency?